source/GlassView.Benchmark/OtherTestBenchmark.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+
+namespace Atmoos.GlassView.Benchmark;
+
+[ShortRunJob]
+[IterationCount(7)]
+[MemoryDiagnoser]
+public class OtherTestBenchmark
+{
+    private const Int32 count = 8096;
+
+    [Benchmark(Baseline = true), BenchmarkCategory("SomeCat")]
+    public void SumStrings() => Thread.Sleep(7);
```

Comment:
Why do we have here `Thread.Sleep` ?

Reply:
Quite right. This was all silly stuff. I've fixed this by:

- Marking all benchmarks as dummies
- Doing something that's consistent within a benchmark class
- Establishing three "Categories": Simple, Regular and Complex benchmarks.